src/views/clusteroverview/OverviewTab/resource-counts-card/ResourceCountsCard.scss

```diff
@@ -0,0 +1,30 @@
+.resource-counts-card {
+  margin-bottom: 1rem;
+
+  &__grid {
+    grid-auto-flow: column;
+    grid-auto-columns: 1fr;
+  }
+
+  &__resource {
+    display: inline-block;
+  }
+
+  &__grid-item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 24px;
+  }
+
+  &__count-value {
+    color: var(--co-global--palette--blue-400);
+    font-size: var(--pf-global--FontSize--4xl);
+    text-align: center;
+    margin-bottom: -10px;
+  }
+
+  &__count-name {
+    text-align: center;
+  }
+}
```

Comment:
can we use var values here? instead of absolute numbers?

Reply:
Fixed!